CHANGELOG.md

```diff
@@ -0,0 +1,27 @@
+## 1.1.0
+- Upgraded dependencies to PSR-17
+
+**Breaking Changes:**
+- Changed type of `errors` in `PaynowException` to `Error`
```

Comment:
If you are introducing BC then new lib version should be 2.0.0

Reply:
you are welcome guys